shared/src/main/kotlin/com/egm/stellio/shared/util/ApiUtils.kt

```diff
@@ -66,16 +63,63 @@ fun extractContextFromLinkHeader(req: ServerRequest): String {
         NgsiLdParsingUtils.NGSILD_CORE_CONTEXT
 }
 
+fun List<MediaType>.isAcceptable(): Boolean {
+    return this.map {
+        it == MediaType("*", "*") ||
+        it == MediaType("application", "*") ||
+        it == MediaType("application", "json") ||
+        it == MediaType("application", "ld+json")
+    }.contains(true)
+}
+
+// TODO: add check for allowed methods and return 405
```

Comment:
In my last project we were using spring annotation to configure routes. https://www.baeldung.com/spring-requestmapping
then to apply pre/post request / error handling we were working with Interceptor / Aspects

Reply:
i saw that spring annotations offers a simple way to declare supported methods.
since we are configuring our routes differently, i implemented it this way: 

https://github.com/stellio-hub/stellio-context-broker/pull/40/commits/c0fb0cf80db48e31acf9462dd292620b94a04e88